Source/Atomic/UI/UISelectList.h

```diff
@@ -70,6 +70,11 @@ class UISelectList : public UIWidget
 
     void SetUIListView(bool value);
 
+    /// Returns the string of item at the requested index
+    const String& GetItemString(int index);
+    /// Returns the string of the selected item
```

Comment:
I think we can change these signatures to return a String by value, as the common case is creating a new String, copying that to a member variable, and returning it to satisfy returning a reference... We might as well just return a new String by returning the const char* directly which will be automatically be converted.  This avoids the introduction of a new member variable and the copy,  when there isn't an item string, we can still return String::EMPTY